### PR TITLE
Add target option

### PR DESCRIPTION
**Pull request by expatiating**
Fixes issue #6

### CODE REVIEW

**Comment by expatiating** [comment]
@jhuleatt My failing step is `Error: Input required and not supplied: firebaseServiceAccount` I don't believe that I manage this in my PR. Happy to help to fix, but I'm not sure if I'd have access.

File: action.yml
Comment (jhuleatt) [inline action.yml:50]: @rachelsaunders can you please give this description a quick review? I want to make sure we get the deploy target terminology correct.

File: src/deploy.ts
Comment (jhuleatt) [inline src/deploy.ts:135]:
```suggestion
    ["deploy", "--only", `hosting${target ? ":" + target : ""}`],
```

Suggested change since we need `--only hosting` in both cases

File: src/deploy.ts
Comment (jhuleatt) [inline src/deploy.ts:52]: Should this be optional (`target?: string`) since it won't always be defined?

File: src/deploy.ts
Comment (jhuleatt) [inline src/deploy.ts:113]:
```suggestion
      ...(target ? ["--only", target] : []),
```

(to match the `expires` line below)

**Review by jhuleatt — CHANGES_REQUESTED** [review]
Thank you @expatiating! I've added a few comments. Once those are resolved, we can get this merged!

File: action.yml
Comment (rachelsaunders) [inline action.yml:50]:
```suggestion
      "The target name of the Hosting site to deploy to. If you leave this blank,
      the default target or all targets defined in the .firebaserc will be deployed to.
      Refer to the Hosting docs about [multiple sites](https://firebase.google.com/docs/hosting/multisites)
      for more information about deploy targets."
```

**Review by rachelsaunders — CHANGES_REQUESTED** [review]

File: action.yml
Comment (jhuleatt) [reply action.yml:50]: Thanks!

File: src/deploy.ts
Comment (expatiating) [reply src/deploy.ts:52]: Agree, should be optional

**Review by expatiating — COMMENTED** [review]
@jhuleatt and @rachelsaunders thank you for the review.

I believe I've addressed all comments and committed all suggestions.

**Review by jhuleatt — APPROVED** [review]

**Comment by jhuleatt** [comment]
Thank you @expatiating! This is now available in [`v0.5-alpha`](https://github.com/FirebaseExtended/action-hosting-deploy/releases/tag/v0.5-alpha). You can try it with the following yml config:

```yaml
- name: Deploy to Firebase Hosting
  uses: FirebaseExtended/action-hosting-deploy@v0.5-alpha
```

I'll probably wait until early next week to release to the `@v0` tag, which makes it available to everyone that set up this action via the CLI. Just want to avoid a Friday release.

**Comment by expatiating** [comment]
@jhuleatt just migrated my workflow to [`v0.5-alpha`](https://github.com/FirebaseExtended/action-hosting-deploy/releases/tag/v0.5-alpha), restored the changes I made to my `firebase.json` / `.firebaserc` to work with [`v0`](https://github.com/FirebaseExtended/action-hosting-deploy/releases/tag/v0) and voila, worked like a charm! 

Thank you and glad that we made this change! 😀

**Pull request merged**
